datastore/longhorn.go

```diff
@@ -4092,6 +4104,21 @@ func (s *DataStore) ListSystemBackupsRO() ([]*longhorn.SystemBackup, error) {
 	return s.sbLister.SystemBackups(s.namespace).List(labels.Everything())
 }
 
+func TagSystemBackupVersionLabel(version string, obj runtime.Object) error {
```

Comment:
This is only used for system backup controller? why not just move to it and make it as a private function.

Suggest naming `labelSystemBackupLonghornVersion`.

Reply:
The convention is not quite clear for me, but it's good now.